jax/experimental/jax2tf/jax2tf.py

```diff
@@ -539,7 +551,12 @@ def process_call(self, call_primitive: core.Primitive, f: lu.WrappedFun,
     assert call_primitive.multiple_results
     vals: Sequence[TfVal] = [t.val for t in tracers]
     f = _interpret_subtrace(f, self.main, tuple(t.aval for t in tracers))
-    vals_out: Sequence[Tuple[TfVal, core.AbstractValue]] = f.call_wrapped(*vals)
+    if call_primitive.name == "named_call":
+      with tf.name_scope(_sanitize_scope_name(params["name"])):
+        vals_out: Sequence[Tuple[TfVal,
```

Comment:
Yes, I did a bisect and find after https://github.com/google/jax/pull/4470/, the name_scope of forward pass (which was missing) show up in xprof. Before that, the jax2tf process_call function doesn't capture the named_call of the forward pass, after the PR, those start to appear. 

The named_call function does have an argument, I'm not sure why that will be constant folded by jax prior to the conversion. (Maybe because that doesn't have an effect on the output?)